### PR TITLE
Fix up docs and format issues

### PR DESCRIPTION
**Pull request by umohnani8**
Clean up some format issues and also fix the command to build the docs so that is built accurately and in the right location for it to show up in readthedocs.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: podman/api/ssh.py [hunk 1]
```diff
@@ -202,6 +203,7 @@ def connect(self) -> None:
 class SSHConnectionPool(urllib3.HTTPConnectionPool):
     """Specialized HTTPConnectionPool for holding SSH connections."""
 
+    # pylint: disable=invalid-name
     ConnectionCls = SSHConnection
```
Comment (jwhonce) [inline podman/api/ssh.py:207]:
```suggestion
    ConnectionCls = SSHConnection     # pylint: disable=invalid-name
```
Should disable the error for just this line, not the whole file.

File: podman/api/uds.py [hunk 2]
```diff
@@ -89,6 +90,7 @@ def connect(self) -> None:
 class UDSConnectionPool(urllib3.HTTPConnectionPool):
     """Specialization of HTTPConnectionPool for holding UNIX domain sockets."""
 
+    # pylint: disable=invalid-name
     ConnectionCls = UDSConnection
```
Comment (jwhonce) [inline podman/api/uds.py:94]:
```suggestion
    ConnectionCls = UDSConnection     # pylint: disable=invalid-name

```

**Review by jwhonce — COMMENTED** [review]
Nits.

LGTM

File: podman/api/client.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import requests
 from requests.adapters import HTTPAdapter
 
-from podman import api
+from podman import api  # pylint: disable=cyclic-import
```
Comment (umohnani8) [inline podman/api/client.py:10]: @jwhonce this fixes the cyclic import error we were getting. I think this should be fine as we are only using `api` to get the version number so I don't think having a cyclic import here should break anything. But not entirely sure.

Comment (jwhonce) [reply podman/api/client.py:10]: Ack, shouldn't be a problem...

File: podman/api/ssh.py (hunk 1 above)
Comment (umohnani8) [reply podman/api/ssh.py]: Fixed

File: podman/api/uds.py (hunk 2 above)
Comment (umohnani8) [reply podman/api/uds.py]: Fixed

**Comment by umohnani8** [comment]
@cevich this fixes the cyclic import error and should help get the tests in your PR passing once in.
Can I please get a lgtm here @lsm5 @rhatdan @cevich

**Review by cevich — APPROVED** [review]
LGTM - Goodness this is big.  Thanks Urvashi!

**Comment by lsm5** [comment]
@umohnani8 LGTM. Do you wanna squash commits or did you intend for them to be 3 separate commits?

**Comment by umohnani8** [comment]
I was planning on leaving them as 3 separate commits @lsm5

**Pull request merged**